Services.Test/DiagnosticsLoggerTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Diagnostics;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Http;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Runtime;
+using Moq;
+using Services.Test.helpers;
+using Xunit;
+
+namespace Services.Test
```

Comment:
the folder structure of test classes should match that of the classes under test. This convention makes it easy to find code and tests, and also allows to organize tests logically (as opposed to have a long list of files in the same folder).
In this case, there should be a `Diagnostics` folder, containing this class, and the namespace should be `Services.Test.Diagnostics`.

Please check also the other files if they are in the right folder